azuredevops/models.go

```diff
@@ -32,19 +33,19 @@ type ApiResourceLocation struct {
 
 // WrappedImproperError
 type WrappedImproperError struct {
-	Count *int           `json:"count,omitEmpty"`
-	Value *ImproperError `json:"value,omitEmpty"`
+	Count *int           `json:"count,omitempty"`
+	Value *ImproperError `json:"value,omitempty"`
 }
 
 // ImproperError
 type ImproperError struct {
-	Message *string `json:"Message,omitEmpty"`
+	Message *string `json:"Message,omitempty"`
 }
 
 // KeyValuePair
 type KeyValuePair struct {
-	Key   *interface{} `json:"key,omitEmpty"`
-	Value *interface{} `json:"value,omitEmpty"`
+	Key   *interface{} `json:"key,cFv"`
```

Comment:
@bvwells , thanks for your contribution!  Can you point me to some documentation on the meaning of `cFv` in the json struct tag?  I couldn't find any information about it.

https://golang.org/pkg/encoding/json/#Marshal

Reply:
@tedchamb - thanks for the review. That's just me being stupid, should have double checked the PR before submitting. :-)

Pushed an update.